ghcide/exe/Main.hs

```diff
@@ -83,18 +85,23 @@ main = do
     whenJust argsCwd IO.setCurrentDirectory
 
     dir <- IO.getCurrentDirectory
-    command <- makeLspCommandId "typesignature.add"
 
-    let plugins = Completions.plugin <> CodeAction.plugin
+    let hlsPlugins = pluginDescToIdePlugins [GhcIde.descriptor "ghcide"]
```

Comment:
/cc @gdevanla you can add HLS plugin descriptors to this list and they will be loaded in ghcide